sdk/src/main/scala/com/horizen/account/state/EoaMessageProcessor.scala

```diff
@@ -0,0 +1,44 @@
+package com.horizen.account.state
+
+import scorex.util.ScorexLogging
+
+import java.math.BigInteger
+import scala.util.Failure
+
+/*
+ * EoaMessageProcessor is responsible for management of regular coin transfers inside sidechain.
+ * In our case to make a transfer from one user account (EOA account) to another user account.
+ */
+object EoaMessageProcessor extends MessageProcessor with ScorexLogging {
+  //TODO: actual gas to be defined
+  val gasUsed: BigInteger = BigInteger.valueOf(20000L)
+
+  override def init(view: AccountStateView): Unit = {
+    // No actions required for transferring coins during genesis state initialization.
+  }
+
+  override def canProcess(msg: Message, view: AccountStateView): Boolean = {
+    // Can process only EOA to EOA transfer, so when:
+    // 1. "from" account must be an EOA account - no "code" set;
+    // 2. transaction "data" is empty.
+    msg.getData.isEmpty && view.getCodeHash(msg.getFrom.address()).isEmpty
```

Comment:
I just checked: A TX with data is still a valid EOA to EOA transaction in Ethereum. When the `to` address has no code the data is ignored. It is just very uncommon because adding data increases the intrinsic gas required for the transaction.

Reply:
I think we could decide not to support that, but to be as compatible as possible we should allow data and ignore it.

Reply:
good, thanks. Let's be more compatible: so it is allowed but ignored.
I will update the spec